frontend/src/container/ListOfDashboard/SearchFilter/index.tsx

```diff
@@ -0,0 +1,216 @@
+import { CloseCircleFilled } from '@ant-design/icons';
+import { useMachine } from '@xstate/react';
+import { Button, Select } from 'antd';
+import { RefSelectProps } from 'antd/lib/select';
+import history from 'lib/history';
+import { filter, map } from 'lodash-es';
```

Comment:
please import via 
`import filter from "lodash-es/filter"`

also we have native filter via js